features/Context/FieldTypeFormContext.php

```diff
@@ -144,23 +145,120 @@ public function theFieldDefinitionIsMarkedAsRequired()
      */
     public function theInputFieldsShouldBeFlaggedAsRequired()
     {
-        $inputNodeElements = $this->getSession()->getPage()->findAll(
+        $this->theFieldShouldBeFlaggedAsRequired('input');
+    }
+
+    /**
+     * @Given /^the field definition is set to single choice$/
```

Comment:
No problem. You've quoted my favourite XKCD comic, I can't argue against that :-)